README.md

```diff
@@ -37,6 +37,43 @@ AI tournaments). Changed defaults will be advertised when the match begins.
  - If you would like to develop a bot in Java, please follow [this tutorial](http://sscaitournament.com/index.php?action=tutorial).
  - Want to help improve BWAPI? Fork the [`develop` branch](https://github.com/bwapi/bwapi/tree/develop).
 
+## Quick Start for 4.1.2 {#quick-start-for-4.1.2}
```

Comment:
Don't specify the version. This creates a dependency that developers will have to update for every single release. Just call it "Quick Start".